core/src/main/java/org/apache/accumulo/core/metrics/MetricsProducer.java

```diff
@@ -597,6 +639,18 @@ public interface MetricsProducer {
   String METRICS_LOW_MEMORY = "accumulo.detected.low.memory";
   String METRICS_COMPACTOR_PREFIX = "accumulo.compactor.";
   String METRICS_COMPACTOR_MAJC_STUCK = METRICS_COMPACTOR_PREFIX + "majc.stuck";
+  String METRICS_COMPACTOR_JOB_PRIORITY_QUEUES = METRICS_COMPACTOR_PREFIX + "cjpq.count";
```

Comment:
I wonder if we should use `queue` instead of `cjpq`.

Reply:
Does the naming change between `queue` and `group` in #3591 impact these names as well? 

If not, I'm happy to swap this to `queue`.

Reply:
`queue` is correct here as these are metrics for the priority queue in the CompactionCoordinator - a data structure. The `group` name is for the concept of resource groups, which has to do with processes (compactor, manager, tserver, etc.)